pkg/dsl/definition/template.go

```diff
@@ -29,6 +29,12 @@ const (
 	PatchFieldName = "patch"
 )
 
+const (
+	// ExtraWorkloadObj defines the extra workload obj from a workloadDefinition,
+	// e.g. a workload composed by deployment and service, the service will be marked as ExtraWorkloadObj
+	ExtraWorkloadObj = "ExtraWorkloadObj"
```

Comment:
How about we using `CompoundWorkloadObj`?

Reply:
No， ExtraWorkloadObj is better

Reply:
How about "AuxilaryWorkload", extra makes it sound like unnecessary

Reply:
AuxiliaryWorkload is good!